src/shared/components/drawer/drawer.less

```diff
@@ -103,6 +103,13 @@
 		padding: 0px 10px;
 	}
 
+	ul {
+		li {
+			margin: 0px;
+			padding: 0px;
```

Comment:
I'm not sure why we are going to remove padding here, having padding to simulate indentation should help to understand "tree" structure